src/api/treeTrackerApi.js

```diff
@@ -137,35 +137,37 @@ export default {
       handleError(error);
     }
   },
-  getCaptureCount(filter) {
+  /**
+   * Captures
+   */
+  getCaptures({ ...params }) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/count?where=${JSON.stringify(
-        filter.getWhereObj()
-      )}`;
+      let filterObj = { limit: 25, offset: 0, ...params };
```

Comment:
**Question:** Any specific reason to limit this to 25? 🤔

Reply:
Not sure how this got in there. The API defaults to 25 if no limit is specified, so this is unnecessary.
Now removed.